components/core/src/clp_s/CommandLineArguments.cpp

```diff
@@ -202,6 +202,11 @@ CommandLineArguments::parse_arguments(int argc, char const** argv) {
                     "disable-log-order",
                     po::bool_switch(&m_disable_log_order),
                     "Do not record log order at ingestion time."
+            )(
+                    "file-type",
+                    po::value<std::string>(&m_file_type)->value_name("FILE_TYPE")->
```

Comment:
For the string values that this accepts I'm not sure if I like `Json` and `IR` -- both because it would be the only option that forces users to enter a capitalized option, and because IR can ambiguously be referring to ir or irv2.

How about `jsonl` and `kv-pair-ir`?

Reply:
I don't have a strong opinion, but I do tend to like shorter rather than longer. How about `kv-ir` instead of `kv-pair-ir`.  I personally would even do just `json` instead of `jsonl`.  What do you think?

Reply:
`kv-ir` and `json` both sound good to me.